spec/books/user_visits_a_page_for_a_book_spec.rb

```diff
@@ -0,0 +1,99 @@
+                          user: user1)
+      review = Review.create(rating: 4,
+                          body: "What is this book about again?",
+                          book: book,
+                          user: user2)
+
+      visit "/books/#{book.id}"
+
+      expect(page).to have_content("Average Rating: 4")
+    end
+
+    it "the user sees the highest rating for that book" do
+      book = Book.create(title: "Old Man and the Sea")
+      user = User.create(name: "John")
+      user1 = User.create(name: "Hank")
+      user2 = User.create(name: "Holly")
+      review = Review.create(rating: 5,
+                          body: "A Charming Tale",
+                          book: book,
+                          user: user)
+      review = Review.create(rating: 3,
+                          body: "A Charmingish Tale",
+                          book: book,
+                          user: user1)
+      review = Review.create(rating: 4,
+                          body: "What is this book about again?",
+                          book: book,
+                          user: user2)
+
+      visit "/books/#{book.id}"
+
+      expect(page).to have_content("Highest Rating: 5")
+    end
+
+    it "the user sees the lowest rating for that book" do
+      book = Book.create(title: "Old Man and the Sea")
+      user = User.create(name: "John")
+      user1 = User.create(name: "Hank")
+      user2 = User.create(name: "Holly")
+      review = Review.create(rating: 5,
+                          body: "A Charming Tale",
+                          book: book,
+                          user: user)
+      review = Review.create(rating: 3,
+                          body: "A Charmingish Tale",
+                          book: book,
+                          user: user1)
+      review = Review.create(rating: 4,
+                          body: "What is this book about again?",
+                          book: book,
+                          user: user2)
+
+      visit "/books/#{book.id}"
+
+      expect(page).to have_content("Lowest Rating: 3")
+    end
+
+    it "the user sees the lowest rating for that book" do
+      book = Book.create(title: "Old Man and the Sea")
+      user = User.create(name: "John")
+      user1 = User.create(name: "Hank")
+      user2 = User.create(name: "Holly")
+      review = Review.create(rating: 5,
+                          body: "A Charming Tale",
+                          book: book,
+                          user: user)
+      review = Review.create(rating: 3,
+                          body: "A Charmingish Tale",
+                          book: book,
+                          user: user1)
+      review = Review.create(rating: 4,
+                          body: "What is this book about again?",
+                          book: book,
+                          user: user2)
+
+      visit "/books/#{book.id}"
+
+      within ("high-rating") do
+          expect(page).to have_content("A Charming Tale")
+      end
```

Comment:
This is likey not working because you didn't tell it to look for a class (.higest-rating) or id (#highest-rating).